lib/health_check/custom_health_checks.rb

```diff
@@ -0,0 +1,50 @@
+module HealthCheck
+  class CustomHealthChecks
+
+    def self.check_sidekiq_redis
+      Sidekiq.redis { |r| "" if r.ping == "PONG" }
+    rescue => e
```

Comment:
I am not sure if here shouldn't you use `rescue Exception` like in `check_redis` method.